docs/index.asciidoc

```diff
@@ -57,9 +60,9 @@ filter plugins.
   * Value type is <<string,string>>
   * There is no default value for this setting.
 
-The cipher algorithm
+The cipher algorithm to use for encryption and decryption operations.
 
-A list of supported algorithms can be obtained by
+A list of supported algorithms depends on the versions of Logstash, JRuby, and Java this plugin is running in, but can be obtained by
 [source,ruby]
     puts OpenSSL::Cipher.ciphers
```

Comment:
@karen this combines your input, and is also even more helpful because it references a command they can actually run:

```suggestion
A list of supported algorithms depends on the versions of Logstash, JRuby, and Java this plugin is running in, but can be obtained by running:
[source,sh]
    cd $LOGSTASH_HOME # <-- your Logstash distribution root
    bin/ruby -ropenssl -e 'puts OpenSSL::Cipher.ciphers'
```